### PR TITLE
Adds new sound effects for meat hook

### PR DESCRIPTION
**Pull request by Tenryumi**
A minor change to make shooting a meat hook feel like it has a lot more weight.
Previously it used the baton-extend sound effect... really? 😕

Now the meat hook has its own unique, distinct noise, changed to a sound that I think is more...appropriate.
If you were curious what the new sound is, you can listen to it [here](http://noproblo.dayjo.org/ZeldaSounds/OOT/OOT_Hookshot_Fire.wav).
As for the other sound efect, it should go without saying what it is.

### CODE REVIEW

**Comment by Tenryumi** [comment]
TRAVIS PLS NO WE CAN TALK THIS OUT C'MON

**Comment by ShadowDeath6** [comment]
> not making it play "GET OVER HERE"

Smhbbq

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -172,7 +172,7 @@
 	ammo_type = /obj/item/ammo_casing/magic/hook
 	icon_state = "hook"
 	item_state = "chain"
-	fire_sound = 'sound/weapons/batonextend.ogg'
+	fire_sound = 'sound/weapons/meathook_shoot.ogg'
```
Comment (ShadowDeath6) [inline code/modules/mining/lavaland/necropolis_chests.dm]: meathook_fire.ogg

**Comment by Tenryumi** [comment]
Better idea. Why not make that the new _hit sound?_
Thank you @ShadowDeath6  for reminding me of the existence of Mortal Kombat. Implementing this at once.

**Comment by Tenryumi** [comment]
I accidentally blasted my own ears  with Sorpion's scream while editing and converting that sound effect.

**Comment by Cruix** [comment]
Is that sound effect copyrighted?

**Comment by Tenryumi** [comment]
I'd imagine not. The game the sound originates from is well over 10 years old, and there are thousands of media instances (videos, music, movies, etc) that use that signature line.
[I'm looking at a Youtube video that's 6 years old that uses every single possible iteration of the line from multiple games at the time it was made](https://youtu.be/NwJgWIurejQ), and it has had no copyright strikes whatsoever.

**Pull request merged**